### DISCUSSION TITLE
Docker debug command/similar functionality support in Rancher desktop

### DISCUSSION THREAD

**Discussion by sureshg** [rancher-sandbox/rancher-desktop#6769] [Q&A] [answered]
Does Rancher desktop (using dockerd container engine) support for debugging distroless images like in https://docs.docker.com/reference/cli/docker/debug/ ?

**Comment by jandubois** [comment, marked as answer]
The `docker-debug` plugin is proprietary docker software that only works for Docker Desktop.

You can use https://github.com/iximiuz/cdebug as an open-source alternative.
